lms/templates/verify_student/intro_step.underscore

```diff
@@ -39,9 +39,9 @@
           <% if ( requirements['photo-id-required'] ) { %>
           <li class="req req-1 req-id">
             <h4 class="title"><%- gettext( "Photo ID" ) %></h4>
-            <div class="placeholder-art">
-              <i class="icon-list-alt icon-under"></i>
-              <i class="icon-user icon-over"></i>
+            <div class="placeholder-art fa-stack fa-lg">
+              <i class="icon fa fa-list-alt fa-stack-2x"></i>
+              <i class="icon fa fa-user fa-stack-1x"></i>
```

Comment:
This block isn't displaying as intended
![screen shot 2015-01-09 at 9 51 31 am](https://cloud.githubusercontent.com/assets/2052185/5681298/548c843c-97e5-11e4-919e-4b9756cc40f3.png)